dist/tools/insufficient_memory/add_insufficient_memory_board.sh

```diff
@@ -7,14 +7,16 @@
 # directory for more details.
 #
 
-MAKE_ARGS="-j4"
+if nproc > /dev/null 2>&1; then
+    MAKE_ARGS="-j$(nproc)"
```

Comment:
might as well just put `-j` there, the idea behind the `-j4` was that it could run in the background without interfering with the rest of the system too much

Reply:
With `-j` the number of concurrent jobs is unlimited, but with `-j$(nproc)` it will be bound to the number of processors. In the unbound case the memory pressure can be significant, especially when rust stuff is build as well (I think cargo now honors the job control policy of `make`).

I would like to keep it bound to avoid OOM situations. My assumption is that people spending money on high core count CPUs will also spend money on enough RAM to keep the cores busy, so that this bound is a sane trade-off between RAM use and speed-up.